package.json

```diff
@@ -2,9 +2,11 @@
   "name": "frostflake",
   "version": "0.9.3",
   "description": "FrostFlake is a canvas-rendering game engine written in JavaScript ES6. I originally started FrostFlake in 2014 as a way to have fun while learning the deeper aspects of JavaScript. The goal was to build a simple game engine with the fewest dependencies possible. I rewrote the engine in TypeScript in 2017, and again in ES6 in 2020.",
-  "main": "FrostFlake.js",
+  "main": "/build/FrostFlake.js",
   "scripts": {
-    "build": "webpack"
+    "build": "webpack",
```

Comment:
@profexorgeek what is webpack used for in this lib? Should bundling the the responsibility of the application that is including this lib?

Reply:
I have it working in webpack and I am not sure that the typescript compiler can do this. Let's leave it for now